### ISSUE TITLE
Json to Yaml leaves quotes around keys

### ISSUE THREAD

**Issue by ethanabrooks** [mikefarah/yq#623]
**Describe the bug**
Parsing a json file leaves quotes around keys.

Version of yq: 4.1.0
Operating system: mac
Installed via: homebrew

**Input JSON**
sample.json:
```json
{"a":"Easy! as one two three","b":{"c":2,"d":[3,4]}}
```

**Command**
```
yq eval '.. style= ""' sample.json
```

**Actual behavior**

```yaml
"a": Easy! as one two three
"b":
  "c": 2
  "d":
    - 3
    - 4
```

**Expected behavior** (per https://mikefarah.gitbook.io/yq/usage/convert#json-to-yaml)

```yaml
a: Easy! as one two three
b: # look ma, no quotes
  c: 2
  d:
  - 3
  - 4
```

**Comment by isaacl** [comment]
It'd be nice to have a shorthand for converting from json / fully resetting style... In any case, I wrote this sed script to do some dequoting. It's leaves keys with spaces or weird characters in quotes.
```sh
 sed 's/"\([[:alpha:]][[:alnum:]:_-]*\)": /\1: /'
```

You can also use my awful alias
```sh
alias json2yq="yq e '.. style="'""'"' - | sed 's/"'"\([[:alpha:]][[:alnum:]:_-]*\)": /\1: /'"'"
```

**Comment by mikefarah** [comment]
The bug is a result of the recurse operator (..) only matches map values and not keys. Next release, I'll add some key operators, including a recursive operator that matches key and operators (perhaps ...) , which allow this to work properly, as well as control styling on keys generally.

Fair point re shorthand, I'll look into adding back a flag to fully reset the style.

**Comment by mikefarah** [comment]
Fixed in yq 4.2.0, I've added a new recursive descent triple dots `...` which will match values and map keys.

I've also added a `--prettyPrint/-P` flag as a shorthand - thanks for the suggestion

**Comment by ethanabrooks** [comment]
Looks good. For posterity, that's:
```
yq eval '... style= ""' sample.json
```
or
```
yq eval -P sample.json
```
